person.rb

```diff
@@ -0,0 +1,31 @@
+require './nameable'
+
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
```

Comment:
As requested by the previous reviewers, kindly make sure that the `Person` class inherits from the `Nameable` class.

This is how to implement inheritance:
```ruby
class Person < Nameable
  .
  .

    def correct_name
       name
    end
  .
  .
end
```